cli/tests/test_cmd_doctor_gateway_auth_fail_closed.py

```
@@ -0,0 +1,227 @@
+        pytest.param((0, "connection reset"), "could not be verified", id="transport-failure"),
+        pytest.param((0, "redirect refused"), "could not be verified", id="redirect-refused"),
+        pytest.param((500, "internal error"), "HTTP 500", id="http-500"),
+    ],
+)
+def test_healthy_trusted_gateway_fails_when_status_auth_cannot_be_verified(
+    tmp_path,
+    status_result,
+    expected_detail,
+):
+    """Public health success cannot turn an inconclusive auth probe green."""
+    token = "status-probe-token-must-not-render"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)